extensions/default/src/ViewerLayout/ResizablePanelsHook.tsx

```diff
@@ -270,53 +281,55 @@ const useResizablePanels = (
       return;
     }
     const { width: panelGroupWidth } = resizablePanelGroupElemRef.current.getBoundingClientRect();
-    setRightPanelExpandedWidth((size / 100) * panelGroupWidth - sidePanelExpandedInsideBorderSize);
+    setRightPanelExpandedWidth(
+      (size / 100) * panelGroupWidth - panelGroupDefination.shared.expandedInsideBorderSize
+    );
   }, []);
 
   return [
     {
       expandedWidth: leftPanelExpandedWidth,
-      collapsedWidth: sidePanelCollapsedWidth,
-      collapsedInsideBorderSize: sidePanelCollapsedInsideBorderSize,
-      collapsedOutsideBorderSize: sidePanelCollapsedOutsideBorderSize,
-      expandedInsideBorderSize: sidePanelExpandedInsideBorderSize,
+      collapsedWidth: panelGroupDefination.left.collapsedOffsetWidth,
+      collapsedInsideBorderSize: panelGroupDefination.shared.collapsedInsideBorderSize,
+      collapsedOutsideBorderSize: panelGroupDefination.shared.collapsedOutsideBorderSize,
+      expandedInsideBorderSize: panelGroupDefination.shared.expandedInsideBorderSize,
       onClose: onLeftPanelClose,
       onOpen: onLeftPanelOpen,
     },
     {
       expandedWidth: rightPanelExpandedWidth,
-      collapsedWidth: sidePanelCollapsedWidth,
-      collapsedInsideBorderSize: sidePanelCollapsedInsideBorderSize,
-      collapsedOutsideBorderSize: sidePanelCollapsedOutsideBorderSize,
-      expandedInsideBorderSize: sidePanelExpandedInsideBorderSize,
+      collapsedWidth: panelGroupDefination.right.collapsedOffsetWidth,
+      collapsedInsideBorderSize: panelGroupDefination.shared.collapsedInsideBorderSize,
+      collapsedOutsideBorderSize: panelGroupDefination.shared.collapsedOutsideBorderSize,
+      expandedInsideBorderSize: panelGroupDefination.shared.expandedInsideBorderSize,
       onClose: onRightPanelClose,
       onOpen: onRightPanelOpen,
     },
-    { direction: 'horizontal', id: viewerLayoutResizablePanelGroupId },
+    { direction: 'horizontal', id: panelGroupDefination.groupId },
     {
-      defaultSize: resizablePanelDefaultSize,
-      minSize: resizablePanelDefaultSize,
+      defaultSize: leftResizablePanelDefaultSize,
```

Comment:
Set this to zero. It really is a useless prop for us.

Reply:
Setting this to 0 throws errors in the console, so I'm going to set it to be the same as `minSize`